src/main/java/se/sundsvall/relations/service/mapper/RelationTypeMapper.java

```diff
@@ -0,0 +1,33 @@
+package se.sundsvall.relations.service.mapper;
+
+import org.springframework.stereotype.Component;
+import se.sundsvall.relations.api.model.RelationType;
+import se.sundsvall.relations.integration.db.model.RelationTypeEntity;
+
+@Component
```

Comment:
Doesn't need to be a `@Component`.
Switch to static methods?

Reply:
Component ensures a singelton object. Testing RelationTypeService will be much cleaner since we can verify that the mapper is called and what is returned by the mapper is the same object that is used for further calls. I think it is better to separate and isolate the mapper and avoid static mocking, or the need to verify mapping in service tests.

Reply:
`@Component` are used for classes managed by spring. POJO:s are not managed by Spring.

Static methods does not need static mocking. The static class are tested in a separate class.